src/plugins/exception-helpers.ts

```diff
@@ -11,32 +15,32 @@ export default function exceptionHelpersPlugin (app) {
   app.addHelper('websocketException', websocketException);
 }
 
-function exception (ctx, error) {
+function exception (ctx: MojoContext, error: Error): boolean {
   if (ctx.isWebSocket) return ctx.websocketException(error);
   return ctx.httpException(error);
 }
 
-async function htmlException (ctx, error) {
-  ctx.stash.exception = error;
+async function htmlException (ctx: MojoContext, error: Error): Promise<boolean> {
+  (ctx.stash as MojoStash).exception = error;
```

Comment:
Same here, the type should be inferred from the context object. (Which is not ported yet, is it? Yea that's probably the reason...)